src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -107,6 +109,7 @@ public void beforeEach() {
     StorageTestSuite.deleteAll(itemsStorageUrl(""));
     StorageTestSuite.deleteAll(holdingsStorageUrl(""));
     StorageTestSuite.deleteAll(instancesStorageUrl(""));
+    statisticalCodeFixture.removeTestStatisticalCodes();
```

Comment:
Is this fixture only used by tests?

Reply:
I found it in another test. Will check where else it is used.

Reply:
This fixture is part of TestBase and only used in StatisticalCodeTest and now ItemStorageTest.

Reply:
If you are asking if it is used in multiple tests in ItemStorageTest, then yes it is.

Reply:
I opted for as much isolation as I could achieve. If you want I can create the statistical codes in before all and remove them in after all. Or are they already be removed by some abstraction?

Reply:
Or would you prefer I just move that remove call to each test creating stat codes?

Reply:
I'm not sure what you mean, I haven't mentioned anything about isolation or moving code. 

I assume that method already deletes all statistical codes. The only thing I'd want to do is rename it to remove the redundant `test` from it's name.

Reply:
Oh, I didn't name that method. Sorry for the misunderstanding. Do you think I should rename it in the context of this PR?